src/app/dim-api/dim-api.ts

```diff
@@ -93,7 +93,8 @@ export async function createLoadoutShare(platformMembershipId: string, loadout:
 }
 
 export async function getSharedLoadout(shareId: string) {
-  const params: GetSharedLoadoutRequest = {
+  // https://github.com/microsoft/TypeScript/issues/15300#issuecomment-1320421641
+  const params: Pick<GetSharedLoadoutRequest, keyof GetSharedLoadoutRequest> = {
     shareId,
   };
```

Comment:
alternatively
```ts
  const params = {
    shareId,
  } satisfies GetSharedLoadoutRequest;
```

Reply:
Oh yeah! I had never had occasion to use that before.